core/src/main/java/uk/gov/justice/services/core/jms/JmsSender.java

```diff
@@ -26,8 +33,10 @@ public JmsSender(final Component destinationComponent, final JmsDestinations jms
 
     @Override
     public void send(final JsonEnvelope envelope) {
+        trace(LOGGER, () -> format("Sending JMS message: %s", toEnvelopeTraceString(envelope)));
```

Comment:
Need to add details of where we're sending it - can you get a queue name from the destination?